tflite/python/tflitePythonVivante/Dockerfile

```
@@ -9,30 +9,59 @@ ARG IMAGE_ARCH=
 ##
 # Base container version
 ##
-ARG BASE_VERSION=3.0.8-bookworm
+ARG BASE_VERSION=3.3.0-bookworm
 
 ARG APP_ROOT=
 
-FROM --platform=linux/${IMAGE_ARCH} \
-    torizon/debian:${BASE_VERSION} AS Deploy
+FROM --platform=linux/${IMAGE_ARCH} torizon/debian:${BASE_VERSION} AS Build
 
 ARG IMAGE_ARCH
 ARG APP_EXECUTABLE
 ARG APP_ROOT
+ARG BASE_VERSION
 
-# your regular RUN statements here
-# Install required packages
-RUN apt-get -q -y update && \
-    apt-get -q -y install \
-    python3-minimal \
-    python3-pip \
-    python3-venv \
+## Install Python
+RUN apt-get -y update && apt-get install -y \
+  python3 python3-dev python3-numpy python3-pybind11 \
+  python3-pip python3-setuptools python3-wheel \
 # DO NOT REMOVE THIS LABEL: this is used for VS Code automation
     # __torizon_packages_prod_start__
     # __torizon_packages_prod_end__
 # DO NOT REMOVE THIS LABEL: this is used for VS Code automation
-    && apt-get clean && apt-get autoremove && \
-    rm -rf /var/lib/apt/lists/*
+  && apt-get clean && apt-get autoremove && rm -rf /var/lib/apt/lists/*
+
+## Install build tools
+RUN apt-get -y update && apt-get install -y \
+    cmake build-essential gcc g++ git wget unzip patchelf \
+    autoconf automake libtool curl gfortran && \
+    apt-get clean && apt-get autoremove && rm -rf /var/lib/apt/lists/*
+
+## Install dependencies
+RUN apt-get -y update && apt-get install -y \
+    zlib1g zlib1g-dev libssl-dev \
```

The shared lib in Debian is always a dependency of the `-dev` package, so we can drop the `zlib1g` package.

```suggestion
    zlib1g-dev \
    libssl-dev \
```